corehq/apps/locations/models.py

```diff
@@ -622,8 +622,7 @@ def path(self):
         try:
             return self._path
         except AttributeError:
-            self._path = list(reversed(self.lineage))
-            self._path.append(self._id)
+            self._path = list(self.get_ancestors(include_self=True).location_ids())
```

Comment:
`get_ancestors` was previously being passed `ascending=True` in this commit. does that still need to happen?

Reply:
`path` and `lineage` have opposite ordering, so using the default `ascending=False` means the results will be returned in the reverse order, which is what we want.